core/server/data/schema/fixtures/fixtures.json

```diff
@@ -358,6 +358,20 @@
         {
             "name": "Post",
             "entries": [
+                {
+                    "title": "Koenig Demo Post",
+                    "slug": "koenig-demo-post",
```

Comment:
As discussed we should change this to `v2-demo-post` to lower the likelihood of collisions with existing posts even further.